requirements.txt

```diff
@@ -6,3 +6,4 @@ docutils>=0.14
 mypy>=0.670
 asttokens>=1.1
 Send2Trash>=1.4
+packaging>=20.9
```

Comment:
This won't do, as Debian Buster only has 19.0

Reply:
TLet me see if 19.0 also works. I've just pinned the latest, admittedly a bit too sloppy.